src/sgx/rust_sgxffi/build.rs

```diff
@@ -0,0 +1,29 @@
+extern crate bindgen;
```

Comment:
Copyright header missing

Reply:
Sure, will take care of updating copyright